opencti-platform/opencti-graphql/src/modules/fintelTemplate/fintelTemplate-types.ts

```diff
@@ -0,0 +1,53 @@
+import type { BasicStoreEntity, StoreEntity } from '../../types/store';
+import { STIX_EXT_OCTI } from '../../types/stix-extensions';
+import type { StixObject, StixOpenctiExtensionSDO } from '../../types/stix-common';
+import type { FintelTemplateWidget } from '../../generated/graphql';
+
+export const ENTITY_TYPE_FINTEL_TEMPLATE = 'FintelTemplate';
+
+export interface FintelTemplate {
+  name: string;
+  description: string;
+  settings_types: [string];
+  instance_filters: string;
+  content: string;
+  fintel_template_widgets: [FintelTemplateWidget];
+  start_date: string;
+}
+
+// region Database types
```

Comment:
I do not understand this comment, how is this database types? We are using types generated by graphql

Reply:
BasicStoreEntity and StoreEntity extensions are called database types, in the XXX-types.ts files, we separate the 'database types' region and the 'stix types' region